src/containers/user/UserSettings.js

```diff
@@ -19,30 +20,44 @@ import { leaveCommunity } from '../../actions/communities'
 import { uploadImage } from '../../actions/uploadImage'
 import A from '../../components/A'
 import { formatDate } from '../../util/text'
-import { debounce, find, get, sortBy, throttle, set } from 'lodash'
+import { debounce, find, sortBy, throttle, set } from 'lodash'
+import { get } from 'lodash/fp'
 import ListItemTagInput from '../../components/ListItemTagInput'
 import { denormalizedCurrentUser, hasFeature } from '../../models/currentUser'
 import { avatarUploadSettings, bannerUploadSettings, defaultBanner } from '../../models/person'
 import { openPopup, setupPopupCallback, PROFILE_CONTEXT } from '../../util/auth'
 import { EDITED_USER_SETTINGS, trackEvent } from '../../util/analytics'
 import { preventSpaces } from '../../util/textInput'
 import Icon from '../../components/Icon'
+import { sendGraphqlQueryAddDataToStore } from '../../actions/graphql'
+
+const fetchUserHasDevice = () =>
+  sendGraphqlQueryAddDataToStore('current', `{
```

Comment:
i'd much rather you used `sendGraphqlQuery` as-is rather than this. if you don't like having to specify a subject and id that go unused, the thing to do might be to move them into the options arg and just have the first arg be `query`.